index.html

```diff
@@ -1495,29 +1510,29 @@
             displays</a>.
           </p>
           <p>
-            While there are live <a>PresentationAvailability</a> objects, the
-            <a>user agent</a> MAY <a>monitor the list of available presentation
-            displays</a> continuously, so that pages can use the <a for=
-            "PresentationAvailability">value</a> property of a
-            <a>PresentationAvailability</a> object to offer presentation only
-            when there are available displays. However, the <a>user agent</a>
-            may not support continuous availability monitoring; for example,
-            because of platform or power consumption restrictions. In this case
-            the <a>Promise</a> returned by <code><a for=
+            While the <a>set of presentation availability objects</a> is not
+            empty, the <a>user agent</a> MAY <a>monitor the list of available
+            presentation displays</a> continuously, so that pages can use the
+            <a for="PresentationAvailability">value</a> property of a
+            <code>PresentationAvailability</code> object to offer presentation
+            only when there are available displays. However, the <a>user
+            agent</a> may not support continuous availability monitoring; for
+            example, because of platform or power consumption restrictions. In
+            this case the <a>Promise</a> returned by <code><a for=
             "PresentationRequest">getAvailability</a>()</code> MUST be
```

Comment:
That has not changed, but the "MUST" is already captured in the `getAvailability` algorithm. I would keep things informative here, e.g. with "in this case the Promise returned by getAvailability() is rejected".